res/controllers/Denon-Prime-4-scripts.js

```diff
@@ -1,17 +1,80 @@
 // eslint-disable-next-line-no-var
 var Prime4 = {};
 
+/*
+ * DENON DJ PRIME 4 - CONTROLLER MAPPING FOR MIXXX
+ *
+ * Thank you for using this mapping! Below are a few parameters
+ * you can change if you want to customize anything :)
+ */
+
+// What colour would you like each deck to be?
+// Choose between "red", "yellow", "green", "cyan", "blue", "magenta", or "white".
+const colourOfDeck1 = "white";
+const colourOfDeck2 = "yellow";
+const colourOfDeck3 = "cyan";
+const colourOfDeck4 = "magenta";
```

Comment:
Use arrays instead of differently numbered variables unless you have a good reason.
```suggestion
const deckColors = ["white", // Deck 1
    "yellow",  // Deck 2
    "cyan",    // Deck 3
    "magenta", // Deck 4
];
```

Reply:
Noted. All unnecessary numbered variables have been converted to arrays.